### PR TITLE
[README.*] update/cleanup - keep up with current code … 

### PR DESCRIPTION
**Pull request by un1versal**
@jjd-uk @MartijnKaijser @wsnipex

This prepares these docs for isengard release. Dont think I missed anything, but if I dint please let me know what.

Notable changes here:
1. Binary addons can be compiled separately from kodi, these changes and move procedures towards that.
2. Documented **audio decoders** procedures missing from https://github.com/xbmc/xbmc/pull/6463
3. README.pvr pointed to the pvr compiling portions in README.linux, but that all could be under same the one file only.

Presuming the entry point for compiling audio decoders is same that should be aright.
##### Please provide feedback on below as well..
1. At some point **in future**, we can make a README.audio and add the audio encoders/decoders procedures to that and point to the source repositories (I presume its at notspiff at this stage?)
2. Leave readmes as is now

I haven’t gone that way atm because: 
This works as is now and the **audio encoders/decoders sources are all over the place**

Ideally, A single **audio repo** similar to that which the README.pvr indicates for PVR and shove all audio related encoders/decoders in there. Food for thought maybe?

@hudokkow you comment was lost but that should be better and fix you mention now.

### CODE REVIEW

**Comment by uNiversaI** [comment]
I probably should add the audio decoders to this as well now... not today.

File: docs/README.linux [hunk 1]
```diff
@@ -3,6 +3,8 @@ TOC
 2. Getting the source code
 3. Installing the required libraries and headers
 4. How to compile
+   4.4 Binary addons
+   4.5 Test Suite
```
Comment (Jalle19) [inline docs/README.linux]: s/Suite/suite/

File: docs/README.linux [hunk 2]
```diff
@@ -115,9 +110,33 @@ Tip: To override the location that Kodi is installed, use PREFIX=<path>.
 For example.
 
     $ make install DESTDIR=$HOME/kodi
+	
+-----------------------------------------------------------------------------
+4.4. Binary addons - compile
+-----------------------------------------------------------------------------
+
+From v14 with commit 4090a5f a new API for binary addons is available.
+You can compile all addons or only specific addons by specifying ADDONS="this,pvr,addon this.audio.encoder"
```
Comment (Jalle19) [inline docs/README.linux]: I think the last part would be more clear with an example like `ADDONS="pvr.foo pvr.bar audioencoder.baz`

File: docs/README.linux [hunk 3]
```diff
@@ -115,9 +110,33 @@ Tip: To override the location that Kodi is installed, use PREFIX=<path>.
...
+    pvr.argustv, pvr.demo, pvr.dvblink, pvr.dvbviewer, pvr.filmon, pvr.hts, pvr.iptvsimple, pvr.mediaportal.tvserver,
+    pvr.mythtv, pvr.nextpvr, pvr.njoy, pvr.pctv, pvr.stalker, pvr.vbox, pvr.vdr.vnsi, pvr.vuplus
+
+Audio decoders: 
+    audiodecoder.modplug, audiodecoder.nosefart, audiodecoder.sidplay, audiodecoder.snesapu,
+    audiodecoder.stsound, audiodecoder.timidity, audiodecoder.vgmstream
 
 -----------------------------------------------------------------------------
-4.1. Test Suite
+4.5. Test Suite
```
Comment (Jalle19) [inline docs/README.linux]: Same thing here (lowercase "s")

File: docs/README.pvr [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+1. Getting the source code
+2. How to compile
 
-They can be found here:
-https://github.com/kodi-pvr/
+-----------------------------------------------------------------------------
+1. Getting the source code
+-----------------------------------------------------------------------------
+
+PVR add-ons source is not included in Kodi's tree directly.
```
Comment (Jalle19) [inline docs/README.pvr]: "The source for PVR addons"

**Comment by uNiversaI** [comment]
@Jalle19 :+1:

File: docs/README.linux [hunk 5]
```diff
@@ -190,7 +209,7 @@ in Settings->Videos->Player from "Auto Detect" to "VDPAU".
 6. Uninstalling
 -----------------------------------------------------------------------------
 
-Issue the commands prepending "sudo", if your user doesn't have write permission,
+Issue the commands pre-pending "sudo", if your user doesn't have write permission,
```
Comment (garbear) [inline docs/README.linux]: prepending is correct, but there might be better ways of phrasing this grammatically

Comment (uNiversaI) [inline docs/README.linux]: Should be better now. dammed auto-correct.

**Pull request merged**